harness/engine/profiles.go

```diff
@@ -64,7 +64,7 @@ func Profile1(index int, node *DuskNode, walletPath string) {
 	viper.Set("timeout.timeoutkeepalivetime", 30)
 
 	viper.Set("logger.output", node.Dir+"/dusk")
-	viper.Set("logger.level", "warn")
+	viper.Set("logger.level", "info")
```

Comment:
Is it not better to have more granular logging levels for the test-harness, in case we need to debug?

Reply:
For this PR  it should be fine to change it from `warn` to `info`. For debugging test-harness problems within CI, `debug` level would be must but let's address in the next PR.

Reply:
I don't understand - what is the point of changing it, and why should we open a new PR to change it back? A change like this may have made sense for debugging during this PR, but it doesn't seem like a change that should be permanent.

Reply:
Agreed.  Change removed.